src/components/AssetLogo.tsx

```diff
@@ -32,12 +32,12 @@ interface AssetLogoProps {
 const AssetLogo = ({ asset, size, showTooltip, className }: AssetLogoProps) => (
   <div className={className} data-tip={showTooltip && (asset.name || asset.id)}>
     {asset.logoURI ? (
-      <LogoImage src={asset.logoURI ?? AlephiumLogoSVG} />
+      <LogoImage src={asset.logoURI} />
     ) : asset.id === ALPH.id ? (
       <LogoImage src={AlephiumLogoSVG} />
-    ) : asset.id ? (
-      <AssetLogoPlaceholder size={size}>?</AssetLogoPlaceholder>
-    ) : null}
+    ) : (
+      <Ghost size={size * 0.7} />
```

Comment:
Let's go for 0.6 if you don't mind. I find 0.7 a bit too much (details).

Reply:
Changed in [4b40581](https://github.com/alephium/explorer/pull/191/commits/4b40581417c185fac62ac725e631a754a6c94108)